### PR TITLE
Add Eversign

### PR DESCRIPTION
**Pull request by Brassdonkey**

### CODE REVIEW

File: entries/e/eversign.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Eversign": {
+    "domain": "eversign.com",
+    "url": "https://www.eversign.com/",
```
Comment (Carlgo11) [inline entries/e/eversign.com.json]: The url tag isn't needed.
```suggestion
```

File: img/e/eversign.com.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="32px" height="32px" viewBox="0 0 32 32" enable-background="new 0 0 32 32" xml:space="preserve">  <image id="image0" width="32" height="32" x="0" y="0"
+    href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAMAAABEpIrGAAAAOVBMVEUAAAD3ugf3uQf4uwf2uQf3
+ugf2ugf3ugdBPzMvMzfxtgjutAl4ZSWYeR6MciH1uQfssgr0uAgyMjKuZTy0AAAAB3RSTlMA/fn2
+5/DlcjCqpwAAAJVJREFUOMuVk9kOgzAMBN1zMT1o+f+PrRWVKCRrW+xLUGYcULBFmuAfocEuCbac
+Et4dAsQGEBuMTarVIFgts61nKjy0pL6EVVcBo7DxpyPo/gD0/NULEF5f+UGBcEd4O8Iy1PNvgCt8
+MPDtIsr2TH6bNMIXgeBGEqP0w8UeVhLbvsYtGTdly/O2zwdH5Nbj+9Hhdcf/B99DGAh5eu7BAAAA
+AElFTkSuQmCC" />
+</svg>
```
Comment (Carlgo11) [inline img/e/eversign.com.svg]: Please use a real SVG file with vector graphics.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: img/e/eversign.com.svg (hunk 2 above)
Comment (Carlgo11) [reply img/e/eversign.com.svg]: Here's a proper SVG:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><circle cx="50" cy="50" r="50" fill="#f3bb04"/><path fill="#2d3433" d="M79.9 21.05 76 22.4a232.75 232.75 0 0 0-11.35 4.42c-7.5 3.15-12.88 6.33-15.45 9.1l-.63.65.06 1.9.12 3.75c.05 1.5.05 1.93-.05 1.93a14 14 0 0 1-2.35-1.9l-1.18-1c-.04 0-.07 1.07-.07 2.52a15 15 0 0 1-.1 2.58c-.05 0-.8-.63-1.65-1.4a23.13 23.13 0 0 0-1.65-1.4 44.75 44.75 0 0 0-1.75 5.5 15.5 15.5 0 0 0-.45 5.28c.1 1.77.1 1.84.47 2.94a11.57 11.57 0 0 0 2.36 3.55c.3.25.22.35-4.85 5.5a144.25 144.25 0 0 0-5.13 5.38c-.17.47.28 1 .75.88.13-.03 2.5-2.38 5.28-5.2a142.7 142.7 0 0 1 5.22-5.13c.07 0 .4.25.75.57 2.25 2.16 5.55 2.56 11 1.3 1.35-.3 5.75-2.05 5.75-2.25 0-.04-.68-.74-1.5-1.57a17.75 17.75 0 0 1-1.5-1.63c0-.07.53-.12 1.38-.17 1.5-.07 3.42-.25 3.55-.33a11.25 11.25 0 0 0-1.38-1.55 19.95 19.95 0 0 1-1.45-1.57c0-.15.53-.2 2.63-.25 2.17-.05 4.12-.23 4.62-.38 1.23-.35 4.15-4.72 6-8.87a100.75 100.75 0 0 0 4.7-13.1 397 397 0 0 1 2.52-8.05c1.43-4.47 1.5-4.17-.77-3.35M18.57 64.13c-.82.65-.62 1.25.98 2.82.68.68 1.25 1.27 1.25 1.32a16.25 16.25 0 0 1-1.3 1.43c-1.57 1.6-1.75 2.33-.75 2.8.7.33.95.2 2.45-1.3a7.36 7.36 0 0 1 1.3-1.15 9.5 9.5 0 0 1 1.4 1.25c1.5 1.48 2.2 1.7 2.73.93.47-.76.32-1.06-1.18-2.6a14.75 14.75 0 0 1-1.25-1.33 15.75 15.75 0 0 1 1.28-1.4c1.42-1.45 1.6-1.75 1.27-2.35-.55-1.05-1.25-.87-2.85.75a10.8 10.8 0 0 1-1.4 1.25c-.07 0-.72-.57-1.43-1.25-1.42-1.45-1.87-1.65-2.5-1.17m.15 12.95c-.62.57-.55 1.77.18 2.2.25.15 2.5.17 31.23.17h30.94l.26-.22a1.4 1.4 0 0 0 0-2.25l-.25-.25h-62l-.35.35"/></svg>
```

File: entries/e/eversign.com.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "contact": {
+      "facebook": "eversign",
```
Comment (phallobst) [inline entries/e/eversign.com.json]:
```suggestion
      "facebook": "geteversign",
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/e/eversign.com.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+      "twitter": "geteversign"
```
Comment (ApCoder123) [inline entries/e/eversign.com.json]:
```suggestion
      "twitter": "geteversign",
      "email": "support@eversign.com"
```

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**